pkg/defaultcomponents/defaults.go

```diff
@@ -68,27 +67,28 @@ func Components() (component.Factories, error) {
 		healthcheckextension.NewFactory(),
 		pprofextension.NewFactory(),
 	}
-
 	for _, ext := range factories.Extensions {
 		extensions = append(extensions, ext)
 	}
-
 	factories.Extensions, err = component.MakeExtensionFactoryMap(extensions...)
 	if err != nil {
 		errs = multierr.Append(errs, err)
 	}
 
 	// enable the selected receivers
-	factories.Receivers, err = component.MakeReceiverFactoryMap(
-		prometheusreceiver.NewFactory(),
-		otlpreceiver.NewFactory(),
+	receivers := []component.ReceiverFactory{
 		awsecscontainermetricsreceiver.NewFactory(),
 		awscontainerinsightreceiver.NewFactory(),
 		awsxrayreceiver.NewFactory(),
 		statsdreceiver.NewFactory(),
+		prometheusreceiver.NewFactory(),
 		jaegerreceiver.NewFactory(),
 		zipkinreceiver.NewFactory(),
-	)
+	}
+	for _, rcv := range factories.Receivers {
```

Comment:
factories.Receivers I'm assuming these are default receivers

Reply:
Yeah. This includes the `otlpreceiver`